server/chat-commands.js

```diff
@@ -896,10 +896,9 @@ const commands = {
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
 	],
 
-	'!groupchatuptime': true,
 	groupchatuptime(target, room, user) {
-		if (!this.runBroadcast()) return;
 		if (!room.uptime) return this.errorReply("Can only be used in a groupchat.");
```

Comment:
```suggestion
		if (!room || !room.uptime) return this.errorReply("Can only be used in a groupchat.");
```

Seems like a better way to implement this.

Reply:
The command shouldn't work at all without a room since it's not flagged. I feel like it's pretty obvious that `/groupchatuptime` needs to be used in the groupchat.

Reply:
Yes. `this.errorReply("Can only be used in a groupchat.");` qualifies as "shouldn't work at all", but with a more informative error message.